BUILD.bazel

```diff
@@ -172,6 +172,15 @@ cc_library(
 
 cc_library(
     name = "experimental-opentelemetry",
+    deprecation = "this library is now GA, please use //:opentelemetry instead.",
+    tags = ["manual"],
```

Comment:
What does this do?

Reply:
If you are quizzing me, the answer is I don't know. I am just copying: https://github.com/googleapis/google-cloud-cpp/blob/76f76c115e9762a76a1a89c9aec6d35cca802423/BUILD.bazel#L54-L64

I think the intention of adding the tag to a target is to skip it in builds.

https://github.com/bazelbuild/bazel/issues/2303 suggests the tag only affects tests, though. Running `bazel query ...` includes `//:experimental-opentelemetry`, so `cc_library`'s are not affected.

I am going to leave the useless tag in case we decide to use `--build_tag_filter` in our CI. :shrug: